### ISSUE TITLE
[SSH Keychain widget] Browsing to a configuration file and clicking OK in browse dialog starts the process automatically

### ISSUE THREAD

**Issue by RuthUlloa** [microsoft/devhome#3565]
### Steps to reproduce the bug

1. Pin the SSH Keychain widget
2. Click the "Choose file" button
3. Select the configuration file
4. The widget processes the file automatically instead of populating the file path into the text box, to then pressing Save to perform the action.

### Expected result

Browsing to config file in the SSH keychain widget browse dialog starts the process automatically.

Standard behavior for this type of UI in Windows is that when you browse to a file and click OK it will populate the file path into the text box, then when you press Save it will take the action.

Basically at this moment the "Choose file" and "Save" buttons are performing the same action, when the expected would be the choose file button to change the value of the text box with the file path of the chosen file. And save button to perform the action with that file.

### Actual result

The widget processes the file automatically instead of populating the file path into the text box, to then pressing Save to perform the action.
